resources/xlf/it/mssql.it.xlf

```diff
@@ -1023,7 +1023,7 @@
         <target state="translated">Il file locale {0} non esiste.</target>
       </trans-unit>
       <trans-unit id="sparkJobSubmission">
-        <source xml:lang="en">No Sql Server Big Data Cluster found.</source>
+        <source xml:lang="en">No Sql Server big data cluster found.</source>
         <target state="translated">Non è stato trovato alcun cluster Big Data di SQL Server.</target>
```

Comment:
So a couple of things :

1. Do we actually need to update each separate xlf file or is just updating the English text in the ts file enough to then have the translators see that it's been changed and update the strings accordingly?
2. Do we need to inform the loc team that these have changed or will they do that automatically?
3. I see here that the loc version is "cluster Big Data" which doesn't match the casing there before (Big Data Cluster). Do we need to be really careful for product/feature names and make sure to add a comment to the localize call to tell them that this is a product/feature name and should be treated specially?

Reply:
I'm doing a find & replace all. I believe this is picked up automatically, but not sure?